tests/test_blocks.py

```diff
@@ -592,19 +593,18 @@ def test_synced_block_unshared():
     assert unshared_reference_markdown == ""
 
 
-@pytest.mark.xfail(reason="Not fully implemented")
 def test_link_to_page_page():
-    # TODO: implement these tests; use mock.patch to mockout the notion API's
-    # response to the get_database_or_page call inside the default linktopage thing
-    # Also, create a separate `mock_page` and `mock_database` calls to test.
-    # Follow the format of the data in the Notion API docs for these
-    mock_page_id = mock_id()
-    notion_block = mock_block(
-        "link_to_page", {"type": "page_id", "page_id": mock_page_id}
-    )
-    pandoc_ast, markdown = process_block(notion_block)
-    assert pandoc_ast == Para([Str("All Blocks Test Page")])
-    assert markdown == "All Blocks Test Page\n"
+
+    mock_link_to_page_block = mock_block(
+        "link_to_page", {"type": "page_id", "page_id": mock_id()})
```

Comment:
I haven't gotten flake8 to enforce this style guideline, but in this repo, the trailing paren should be on its own line and there should be a trailing comma too
```suggestion
        "link_to_page", {"type": "page_id", "page_id": mock_id()},
    )
```

Reply:
Okay, great.   Got it.

Reply:
I'm not sure I understand this convention perfectly, it's application this case.   I will ask you about it the next time we talk.

Reply:
I guess it does make it easier to read, even in this simple case.